include/flatbuffers/base.h

```diff
@@ -29,6 +29,8 @@
   #include <functional>
 #endif
 
+#include "flatbuffers/stl_emulation.h"
```

Comment:
My intent was that this `#include` be conditional on `#ifdef FLATBUFFERS_CPP98_STL`, such that the file is generally not needed at all.